[BULK UPDATE] DocuTune - Rebranding links (part 1) (#31292)
dotnet/docs: docs/architecture/containerized-lifecycle/design-develop-containerized-apps/docker-apps-inner-loop-workflow.md

You can specify additional configuration settings in the Dockerfile, depending on the language and framework you're using. For instance, the `ENTRYPOINT` line with `["dotnet", "WebMvcApplication.dll"]` tells Docker to run a .NET application. If you're using the SDK and the .NET CLI (`dotnet CLI`) to build and run the .NET application, this setting would be different. The key point here is that the ENTRYPOINT line and other settings depend on the language and platform you choose for your application.

> [!TIP]

> > For more information about building Docker images for .NET applications, see [Containerize a .NET app](/dotnet/core/docker/build-container).

Reviewer: ```suggestion
> For more information about building Docker images for .NET applications, see [Docker images for ASP.NET Core](/aspnet/core/host-and-deploy/docker/building-net-docker-images).
```

Reviewer: Aren't the architecture book link texts just supposed to show the URL like this?

> > For more information about building Docker images for .NET applications, go to [https://learn.microsoft.com/dotnet/core/docker/building-net-docker-images](/aspnet/core/host-and-deploy/docker/building-net-docker-images).

Reviewer 2: ```suggestion
> For more information about building Docker images for .NET applications, see [Containerize a .NET app](/dotnet/core/docker/build-container).
```
